### ISSUE TITLE
The VTK support of LCC uses Qt4 instead Qt5

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#229]
The following files use Qt4 instead of Qt5:
```
Linear_cell_complex/benchmark/Linear_cell_complex_3/cmake/ACGCommon.cmake
Linear_cell_complex/examples/Linear_cell_complex/CMakeLCCViewerVtk.inc
Linear_cell_complex/examples/Linear_cell_complex/CMakeLists.txt
Linear_cell_complex/examples/Linear_cell_complex/CMakeLCCViewerQt.inc
```

Cc @gdamiand

**Comment by gdamiand** [comment]
I propose to remove the vtk support, and I modified the qt viewer to use now qt5 (in the branch LCC-examples_qt5-gdamiand).

It is ok now, except that the scene is empty; but I don't think the problem comes from my code since the other elements of qglviewer are also not drawn (for example axis or frame rate).

**Comment by lrineau** [comment]
@gdamiand, do you still want to remove the VTK support?

**Comment by gdamiand** [comment]
Yes.
Note that I need some work to double check and test this branch.

**Comment by sloriot** [comment]
@gdamiand Is this issue still relevant?

**Comment by gdamiand** [comment]
Yes; I need some work to finish the modifications.

**Comment by gdamiand** [comment]
Solved by PR #469
